### PR TITLE
import tools before running linter

### PR DESCRIPTION
**Pull request by ScottSuarez**
changes to fix linter..

closes https://github.com/hashicorp/terraform-provider-google/issues/9763
**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by ScottSuarez** [comment]
tested with 
https://pantheon.corp.google.com/cloud-build/builds;region=global/e31eea15-5f66-408c-900f-960761f4c5ab;step=22?project=graphite-docker-images

https://github.com/GoogleCloudPlatform/magic-modules/pull/5101

File: .ci/containers/terraform-tester/test_terraform.sh [hunk 1]
```diff
@@ -66,22 +66,22 @@ curl \
 
 set +e
 
-make
```
Comment (melinath) [inline .ci/containers/terraform-tester/test_terraform.sh:69]: I think we need to keep this line so that we catch code compilation issues as part of linting.

File: mmv1/products/binaryauthorization/api.yaml [hunk 2]
```diff
@@ -109,7 +109,7 @@ objects:
                   # TODO (mbang): Exactly one of asciiArmoredPgpPublicKey or pkixPublicKey must be set
                   description: |
                     A raw PKIX SubjectPublicKeyInfo format public key.
-
```
Comment (melinath) [inline mmv1/products/binaryauthorization/api.yaml:112]: unrelated change snuck in 😱

**Review by melinath — CHANGES_REQUESTED** [review]
LGTM apart from adding back in make

File: mmv1/products/binaryauthorization/api.yaml (hunk 2 above)
Comment (ScottSuarez) [reply mmv1/products/binaryauthorization/api.yaml:112]: just whitespace I think its fine ?

Comment (ScottSuarez) [reply mmv1/products/binaryauthorization/api.yaml]: removed

File: .ci/containers/terraform-tester/test_terraform.sh (hunk 1 above)
Comment (ScottSuarez) [reply .ci/containers/terraform-tester/test_terraform.sh]: added --- oops didn't mean to remove this ! good catch !!

**Review by melinath — APPROVED** [review]
lgtm
